package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@webcomponents/shadycss",
   "version": "1.0.0-rc.2",
-  "description": "Styling helpers for ShadyDOM",
+  "description": "Shadow CSS polyfill",
```

Comment:
Ok, done. Please also merge #80 in this case.

BTW, you could do it yourself, because all my pull requests allow edits from maintainers.